script/bridge/deploy-implementation.sh

```
@@ -0,0 +1,29 @@
+if [ -z "$WALLET_NAME" ]; then
+  echo -e "${RED}Usage: ./script/bridge/deploy-implementation.sh <walletName> [<args...>]${NC}"
+  exit 1
+fi
+
+# Make sure the script is run from the root of the project
+PROJECT_ROOT=$(cd "$(dirname "${BASH_SOURCE[0]}")"/../../ && pwd)
+cd "$PROJECT_ROOT" || exit 1
+
+# Run the script on all chains with SynapseBridge deployment
+# Look within deployments/chainName for SynapseBridge.json
+for chainName in $(ls deployments); do
```

Shellcheck (0.10.0)

[error] Iterating over ls output is fragile. Use globs.

(SC2045)